lib/ansible/utils/collection_loader.py

```diff
@@ -12,6 +12,8 @@
 
 from types import ModuleType
 
+import ansible
```

Comment:
@sivel could you please add a comment here explaining the purpose of this import?

Reply:
Sorry, I tried to comment like 20 times yesterday. It needs removed. It was done in the first iteration, where we were working off of the assumption that it was relative to the ansible install.

fwiw, this is now @abadger's PR